buildtest/menu/build.py

```diff
@@ -9,11 +9,14 @@
 import sys
 
 from buildtest.defaults import (
+    DEFAULT_CONFIG_FILE,
```

Comment:
This line is not needed, its not being used in this file

Reply:
Fixed locally.